cmd/juju/commands/plugin_test.go

```diff
@@ -37,7 +37,14 @@ func (suite *PluginSuite) SetUpTest(c *gc.C) {
 	}
 	suite.FakeJujuXDGDataHomeSuite.SetUpTest(c)
 	suite.oldPath = os.Getenv("PATH")
-	os.Setenv("PATH", "/bin:"+gitjujutesting.HomePath())
+
+	// We have to be careful to leave the default cmds that we need for tests
+	// like "touch" (which its in /usr/bin on mac but in /bin on linux).
+	// We doings this, because we need to add "binaries" from the tmp dir and reduce
+	// tests execution, since we are looking into all paths in $PATH to find juju plugins
+	os.Setenv("PATH", fmt.Sprintf(
+		"/bin:/usr/bin:%s", gitjujutesting.HomePath(),
```

Comment:
Can we create another file `plugin_darwin_test.go` that adds `/usr/bin`. to the PATH.

Reply:
We decided to do `if..else` with checking the environment system, because it might create some side-effects